src/mp_api/routes/thermo/client.py

```diff
@@ -111,3 +111,21 @@ def search_thermo_docs(
             fields=fields,
             **query_params,
         )
+
+    def get_phase_diagram_from_chemsys(self, chemsys: str) -> PhaseDiagram:
+        """
+        Get a phase diagram for a given chemsys.
```

Comment:
I would re-phrase this to:

`Get a pre-computed phase diagram for a given chemsys.`